lib/identity/client.js

```diff
@@ -4,6 +4,7 @@ const fetch = require('isomorphic-fetch')
 const PartialClient = require('./partialClient')
 const { SECRET_UUID, TYPES } = require('../../lib/utils/constants')
 const { GroupMember, Search } = require('../../types')
+const { fileAsUrl } = require('../../browser/helpers')
```

Comment:
we should not pull browser helpers into `/lib` code. 

All of the library code is shared between node and browser targets. All platform specific methods need to live in their respective platform directories.

If there is a generic helper, it can get added to the library helper file